packages/git.rb

```diff
@@ -441,12 +166,20 @@ def self.install
       # git bash completion
       source #{CREW_PREFIX}/share/git-completion/git-completion.bash
     GIT_BASHD_EOF
-    IO.write("#{CREW_DEST_PREFIX}/etc/bash.d/git", @git_bashd_env)
+    File.write("#{CREW_DEST_PREFIX}/etc/bash.d/git", @git_bashd_env)
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
+    Dir.chdir "#{CREW_DEST_PREFIX}/bin" do
```

Comment:
Change to
```
FileUtils.ln_s '../musl/bin/git' "#{CREW_DEST_PREFIX}/bin/git"
```

Reply:
For some reason the chdir is avoiding an error where I get this:
![image](https://user-images.githubusercontent.com/1096701/140378169-763d6838-6ce4-4951-994c-1f38765ae126.png)